director/jobs/templates/jobs_list_job.html

```diff
@@ -0,0 +1,48 @@
+<div class="job has-text-black">
+  <div class="job-column job-summary">
+    <span class="job-id is-block leading-1">{% if job.id %}JOB {{job.id}}{% endif %}</span>
+    <span class="tag is-uppercase job-tag {{job.colour}}">{{job.status}}</span>
+    <div class="job-icon status is-flex">
+      <i data-feather="{{job.icon}}" class="icon flex-shrink-0"></i>
+      <span class="text truncate has-text-weight-bold">{{job.method_label}}</span>
+    </div>
+  </div>
+
+  <div class="job-column job-details flex-grow is-flex flex-col">
+    <header class="job-details-header flex-wrap is-flex mb-2">
+
+      <span class="job-info job-icon job-user is-flex">
+          <img src="https://via.placeholder.com/16" width="16" height="16" alt="" class="user-avatar job-user-image icon">
+          <span class="job-user-name is-uppercase">{{job.creator.username}}</span>
+      </span>
+
+      <span class="job-info job-icon is-flex">
+          <i data-feather="calendar" class="icon flex-shrink-0"></i>
+          <span class="job-user-name is-uppercase">{{job.created|date:"F n, Y g:i A"}}</span>
+      </span>
+
+      <span class="job-info job-icon is-flex">
+          <i data-feather="clock" class="icon flex-shrink-0"></i>
+          <span class="job-user-name is-uppercase">{{job.get_runtime}}</span>
+      </span>
+    </header>
+
+    <p class="job-details-content is-size-6 content">
+    {% if job.log|length > 3 %}
+      {{job.log|length}} messages in log
+    {% else %}
+      {% for log in job.log %}
+        {{log.message}}<br>
+      {% endfor %}
+    {% endif %}</p>
```

Comment:
It's useful to see a preview of the log messages but I feel they currently take up  too much vertical space. Perhaps, don't show at all, or limit to a single line (from the last log entry).

Reply:
I've now removed the *"n messages in log"* & as you suggest have the first sentence of the last log item (truncated after 15 words).